09_timer/timer.rb

```diff
@@ -2,15 +2,22 @@
 
 class Timer
   attr_accessor :seconds
-  def initialize(seconds = 0)
-    @seconds = seconds
+
+  def initialize
+    @seconds = 0
   end
 
   def time_string
-  result = []
-  hour 
-  minutes
-  result.join
+    seconds = @seconds % 60
+    minutes = (@seconds / 60) % 60
+    hours = @seconds / (60**2)
+    "#{padded(hours)}:#{padded(minutes)}:#{padded(seconds)}"
+  end
+
+  def padded(num)
+    return '0' + num.to_s if num < 10
```

Comment:
`"0#{num.to_s}"` will work faster than `'0' + num.to_s` according to benchmarks
also `10` is a magic number :)